release-notes/2024-10/README.md

```diff
@@ -0,0 +1,221 @@
+<!-- This file should be renamed to README.md and placed in the directory for the release. -->
+
+# SystemLink Enterprise release 2024-10 Release Notes
+
+The 2024-10 release for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the release 2024-09 to the release 2024-10
+
+<!-- Optional section to include comments and instructions needed to successfully upgrade from the previous release to the current release. If the only changes needed are already captured in Helm Chart Breaking Changes, this section is not needed. -->
+
+## New Features and Behavior changes
+
+- Privileges and roles changes
+    - "Routines" service role was added
+    - "Routine Maintainer" role was added
+    - Routines "Enable" and "Disable" privileges were added
+    - "Cancel notebook execution" privilege was added. This isn't currently being enforced.
+    - Routine "Modify metadata" privilege was renamed to "Modify routines"
+    - "Notebook Execution" privilege category was renamed to "Scripts" to match the application rename
+- As a result of the privilege changes above, non-admin users that could previously perform operations on Routines may not be able to any longer. These users should either be applied the "Routine Maintainer" role or have a new role created that grants Routine privileges.
+- Routines must now be disabled in order to be updated unless the user has all of enable, disable, and modify privileges.
+- testmonitorservice
```

Comment:
@prestwick FYI, that this documentation is linked to this AzDO PR:
- [Pull Request 802072](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/802072): TestMonitor | Upgrade EF Core to 8.0